elements/lisk-p2p/test/unit/peer/base.ts

```diff
@@ -395,18 +397,14 @@ describe('peer/base', () => {
 						ipAddress: '1.1.1.1',
 						sourceAddress: '12.12.12.12',
 						wsPort: 1111,
-						sharedState: {
-							version: '1.1.1',
-						},
+						sharedState: {},
```

Comment:
No longer including version?

Reply:
now we only have `protocolVersion`, version was to support older nodes. We can remove `version` and other fields in another PR. Issue related to it https://github.com/LiskHQ/lisk-sdk/issues/5419